components/DatePick.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { SetStateAction, Dispatch } from "react";
+import DatePicker from "react-datepicker";
+import "react-datepicker/dist/react-datepicker.css";
+import styled from "styled-components";
+
+type DatePickProps = {
+  dates: string[];
+  onSetDates?: Dispatch<SetStateAction<string[]>>;
+  className?: string;
+  inline: boolean;
+};
+
+export default function DatePick({
+  dates,
+  onSetDates,
+  className,
+  inline,
```

Comment:
du nimmst hier viel entgegen, aber in der Artist.tsx gibst du nur dates und inline mit - ich warte mal ab, ob die Komponente nochmal irgendwo aufgerufen wird ;)

Reply:
Die Komponente wird an zwei Stellen gerendert: In der Artist.tsx und in der newuser.tsx. Je nachdem bekommt sie verschiedene Props.